routing_model/index.html

```diff
@@ -790,6 +850,15 @@ <h2>Process for Adult/Elder promotion and demotion including merge</h2>
     </p>
   </div>
   </div>
+  <button class="collapsible">split_needed</button>
+  <div class="content">
+    <p>This function indicates that we need splitting.<br>
+    The details for the trigger are still in slight flux, but here are some possibilities:
+    <ul>
+      <li> The number of Online adults plus elders is above our min section size of 10 elder plus 90 Adults + buffer
+    </ul>
+    </p>
+  </div>
   <div class="mermaid">
       graph TB
       CheckAndProcessElderChange["StartCheckAndProcessElderMergeChange:<br/>No exit - Needs killed"]
```

Comment:
This is one more name that's getting really out of touch (maybe for another commit, though).

Reply:
Yes, working on this clean up of names PR.